torch/_inductor/lowering.py

```diff
@@ -1672,6 +1673,11 @@ def additional_pointwise_ops(op: torch._ops.OpOverload):
             is_pointwise_use(use, additional_pointwise_ops)
             for use in V.current_node.users
         )
+        has_symbolic_shape = any(
+            free_unbacked_symbols(inp.get_size()) for inp in inputs
+        )
+        has_view_use = any(is_view(use.target) for use in V.current_node.users)
```

Comment:
@eellison , I feel my fix is a bit fragile. Let me know if you have some better ideas.